src/agent/driver-test-runner.js

```diff
@@ -122,7 +135,27 @@ export class DriverTestRunner {
         });
       }
     } else if (atName == 'VoiceOver') {
-      if (settings !== 'defaultMode') {
+      if (settings === 'quickNavOn' || settings === 'arrowQuickKeyNavOn') {
+        await this.pressKeysToChangeSetting(
+          ATKey.sequence(ATKey.chord(ATKey.key('left'), ATKey.key('right'))),
+          'quick nav on'
```

Comment:
Interesting that [support.json's instructions text](https://github.com/w3c/aria-at/blob/625a22e13bd117c4bb8f7ecdb5e97454f9e935ac/tests/support.json#L91) implies a completely different phrase for `arrowQuickKeyNavOn`:

> "If VoiceOver said 'arrow quick key nav on' ...."

I did confirm it does say "Quick Nav on/off" on MacOS 14.5; this may need additional clarification.

Reply:
If we discuss the `support.json` file during today's CG meeting, then we should definitely mention this aspect, too.